llvm-spirv/test/transcoding/BitReversePref.ll

```diff
@@ -0,0 +1,1584 @@
+; RUN: llvm-as < %s -o %t.bc
```

Comment:
Please make the test shorter. You only need a call to the `BitReverse` function.